open-feature/dotnet-sdk-contrib
# feat: Add custom dimensions for metrics

## This PR


- Adding custom dimensions for metrics

### Related Issues


Fixes #118 

### Notes


- Unfortunately, this feature has the same issue as the initial metrics. I cannot assert if the tags are added correctly. See #114 for more info.
- Removed the `otel` library from the unit tests so it will use the same one that is provided in the main project.

### How to test

On the yellow boxes you can see the custom metrics.
<img width="1388" alt="Screenshot 2023-12-22 at 13 16 40" src="https://github.com/open-feature/dotnet-sdk-contrib/assets/2493377/2241da10-40d7-4665-ae92-16b7132f816a">
